docs/CHANGELOG.rst

```diff
@@ -24,6 +24,12 @@ Changed
 - **BACKWARD INCOMPATIBLE:** Remove ``jbrowse-bam-coverage`` process
   step from the ``workflow-accel`` workflow. The bigwig coverage track
   is computed in ``align-bwa-trim`` process instead.
+- Use sample name as the name of the data object in:
+
+  -``alignment-bwa-aln``
```

Comment:
Fix the list so that each process renders in it's own line

Reply:
Thanks, I didn't notice that. I did the same thing in my previous PR so I also fixed that entry.